microsoftgraph/microsoft-graph-explorer-v4
# Fix: Expand/Collapse button on Response preview tab

## Overview

Fixes #618 
- Also improves the editor UX to provide a more consistent look and feel with other editors such as VsCode

### Demo
<img width="49" alt="image" src="https://user-images.githubusercontent.com/18407044/145409329-4267aa15-8079-436c-b24f-ba446f34ca93.png">


## Testing Instructions

* Load Graph Explorer
* Run any query
* Click on the expand/collapse button on the Response preview
* Notice the click is responsive and you do not need to click outside for it to work

## Notes
An accessibility issue arises _'Landmarks should have a unique role or role/label/title (i.e. accessible name) combination_'.
This is an issue with the editor and I am not sure if it will be fixed. Ref to this [conversation](https://gitanswer.com/monaco-editor-two-unlabelled-monaco-status-divs-with-role-complementary-breaks-accessibility-compliance-javascript-863140063).
As a workaround I have disabled the check and I'll follow up with the next `monaco-editor` release
